jacoco/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    base
+    id("jacoco-report-aggregation")
+}
+
+dependencies {
+    jacocoAggregation(projects.ksp)
+    jacocoAggregation(projects.processingCommon)
+    jacocoAggregation(projects.processingTests.kspTests)
+    jacocoAggregation(projects.processingTests.processorTests)
+    jacocoAggregation(projects.processor)
+    jacocoAggregation(projects.runtime)
+}
+
+reporting {
+    reports {
+        val testCodeCoverageReport by creating(JacocoCoverageReport::class) {
+            testType.set(TestSuiteType.UNIT_TEST)
+        }
+    }
+}
+
+tasks.check {
+    dependsOn(tasks.named<JacocoReport>("testCodeCoverageReport"))
```

Comment:
This is based on https://docs.gradle.org/current/samples/sample_jvm_multi_project_with_code_coverage_standalone.html, to use a standalone module just for merging coverage reports in a way that makes Gradle happy.